README.md

```diff
@@ -6,10 +6,14 @@ A sample is built and hosted at `gcr.io/kubernetes1-226021/capd-manager:latest`
 
 ### Building the binaries
 
-Requires go with go modules.
+Requires go 1.12+ with go modules.
 
-* `go build ./cmd/...`
-* `go build ./cmd/`
+```
+export GO111MODULE=on
```

Comment:
We should explain why we need to set this as it is not required for building. I don't work in my GOPATH anymore unless I'm working with something not using go modules so this step is unnecessary for me

```suggestion
# required if `cluster-api-provider-docker` was cloned into $GOPATH
export GO111MODULE=on
```

Reply:
i guess this is true, will update in a bit.